raiden-network/light-client
# Installs cypress and prepares dApp for integration image

**Thank you for submitting this pull request :)**

Fixes #1404  

**Short description**

Introduces infrastructure for e2e tests for the dApp using the integration image.
The scope of this PR is the part of the test setup up to the point we have a working test that reaches the connect screen.

**Definition of Done**


**Steps to manually test the change (dApp)**

This requires the integration image to run.

1.  Go to the `/integration` folder
2. Run `run-e2e.sh` this should run the docker image and start the dApp using this environment setup.
3. You should see Cypress test runner (electron app)
4. Press run tests
5. You should see the tests running and interacting with the connection until showing the "Connect new token" screen.
